playwright-e2e/pages/brain/brain-base-page.ts

```diff
@@ -10,9 +10,4 @@ export abstract class BrainBasePage extends PageBase {
     }
     super(page, BRAIN_BASE_URL);
   }
-
-  async finish(): Promise<void> {
-    await this.page.waitForTimeout(1000);
-    await this.page.getByRole('button', { name: 'Finish' }).click();
-  }
```

Comment:
This func can be reuseable so I moved it to `page-base.ts`.

Reply:
Nice!